internal/terraform/context_plan.go

```diff
@@ -556,6 +566,7 @@ func (c *Context) planGraph(config *configs.Config, prevRunState *states.State,
 			RootVariableValues: opts.SetVariables,
 			Plugins:            c.plugins,
 			Targets:            opts.Targets,
+			ImportTargets:      opts.ImportTargets,
```

Comment:
I  was hoping that simply adding the import targets here would be sufficient. However, while it appears that it imports it for the current plan, it doesn't seem to write it back to the state. I'm not sure why, and would probably have to spend a fair amount of time understanding how the planning and applying stages work better in order to fix it.